.github/workflows/bundled-size.yaml

```diff
@@ -12,3 +12,7 @@ jobs:
         with:
           build-script: "build-rollup"
           compression: "none"
+          # no need to report on files that aren't the interesting ones
+          exclude: "{./dist/es.js,./dist/module.js,./dist/array.full.js,**/*.map,**/node_modules/**}"
```

Comment:
I want one module file atleast please, very useful for PRs like these: https://github.com/PostHog/posthog-js/pull/863

Reply:
(and keeps the _module_ file size in check as well, not just the snippet)